### ISSUE TITLE
Asset Browser: Does not immediately show new assets if Prefab Library "Duplicate to project" operation needed to create a new group

### ISSUE THREAD

**Issue by DragoniteSpam** [YoYoGames/GameMaker-Bugs#8566]
### Description

It may be easiest to see the issue with a video:

https://github.com/user-attachments/assets/23f136d9-d385-4c5c-ad59-bc93257f6069

The assets won't appear if a new folder has to be created, but will if the folder they would go into already exists.

The project has to be restarted, closing and re-opening the asset browser won't fix it.

### Steps To Reproduce

1. Start GameMaker
2. 
3. 
4. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.1100 (Betas)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

None

### Sample Package Attached?

- [x] I have attached my bug package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by alicemoretti** [comment]
This is not fixed on IDE v2024.1300.0.755  Runtime v2024.1300.0.765. 

When doing "Duplicate into project" for an asset from the Prefab Library in a project for the first time, the asset is not added to the asset browser. On closing and reopening the project, a folder for the asset is created but it's empty. Trying to duplicate into project the asset again (or any other asset from the Prefab Library) now works as expected. 

So basically it only works on and after the second attempt and after having reopened the project. 

Reopening the bug. Thank you.

**Comment by stuckie** [comment]
Fixed alongside https://github.com/YoYoGames/GameMaker-Bugs/issues/9522

**Comment by alicemoretti** [comment]
This is still happening as described on  IDE v2024.1300.0.773  Runtime v2024.1300.0.779. Reopening the bug.

**Comment by stuckie** [comment]
I cannot seem to reproduce this.. moving to 24.14 to try and pin it down better.
